src/main/plugin/iso19139.ca.HNAP/loc/eng/schematron-rules-common.xml

```diff
@@ -41,6 +41,7 @@
   <DistributionFormatVersion>Value is required for distribution format - File version</DistributionFormatVersion>
   <DistributionFormatInvalid>Value for distribution format is not valid</DistributionFormatInvalid>
   <OnlineResourceUrl>Value is required for Online Resource URL</OnlineResourceUrl>
+  <OnlineResourceProtocol>Protocol (HTTP, HTTPS, FTP, ESRI REST: Map Service, OGC:WMS, OGC:WFS) is required for Online Resource: </OnlineResourceProtocol>
```

Comment:
@wangf1122 
Can you change this to be similar to SecurityLevel which has something like the following.

`<SecurityLevel>Security User Note is not valid. Valid values are:</SecurityLevel>`

Then it list the values when generating the error.  This would eliminated hardcoding the list of protocols.

Reply:
Code updated. The result is:

![image](https://user-images.githubusercontent.com/74916635/163252078-ad8a4132-df9f-471e-80bd-1afd6c2d2d05.png)